prow/plugins.yaml

```diff
@@ -66,6 +66,8 @@ plugins:
   - hold
   - label
   - lgtm
+  - milestonestatus
```

Comment:
Doesn't hook need to be redeployed? I think plugin validation is very strict and the config reloading will break because these plugins do not exist.

Reply:
Yup, I redeployed as soon as this merged.